site/client/blocks/AdditionalInformation/additional-information.scss

```diff
@@ -46,8 +46,11 @@
     }
 
     &[data-variation="full-width"] {
-        background-color: $primary-color-100;
+        @include sm-only {
```

Comment:
nit: do we need the abbreviation "sm"? Does that stand for "small"?

Reply:
yes it does. What do you have in mind? Spelling it out in full?

Reply:
Eh not sure, I don't know the layout code too well. Just wondering if there's something more descriptive. Like if `sm` is equivalent to a single column, and `md` is the sideBySideColumns and `lg` is sideBySidePlusSideBar, maybe those names would be better? But no clue if that's better or worse, as there's a lot of features there I'm unsure of, and I'm not an expert with scss.

Reply:
@breck7 sorry I overlooked this. It's true it is sometimes slightly inconvenient to have to look up the "definition" of a breakpoint in `variables.scss` and in that way what you're suggesting makes sense. On the other hand, where the sm, md, lg, etc... convention shines is quickly seeing the progression in places where you need to override styles from one breakpoint to the next. Scanning a few symbols vs `sideBySidePlusSideBar` feels faster, but that might also be because I've internalized the meaning of these shortcuts. I might give it a go.